src/toil/toilState.py

```diff
@@ -116,13 +123,14 @@ def _buildToilState(self, jobDesc, jobStore, jobCache=None):
         object. Jobs will be loaded from the cache (which can be downloaded from
         the jobStore in a batch) instead of piecemeal when recursed into.
 
-        :param jobDesc: Object representing a job.
-        :param jobStore: Object inheriting toil.jobStores.abstractJobStore.AbstractJobStore.
-        :param jobCache:
-        :return:
+        :param toil.job.JobDescription jobDesc
```

Comment:
Again these are duplicative with the type hints, although the old comments being replaced here were kind of unhelpful.